### ISSUE TITLE
Add support for feeds

### ISSUE THREAD

**Issue by filoozom** [ethersphere/beeload-action#13]
In order to be able to deploy production websites to Swarm, it would be easy useful for this action to support feeds.

This could be done either by conditionally replacing `swarm-cli upload` with `swarm-cli feed upload` and adding `inputs` for:
- `topic`
- `topic-string`
- `identity`

Or adding a separate action that runs `swarm-cli feed update`.

What would be your preference? I'd be happy to write a PR. 🙂

**Comment by AuHau** [comment]
Yeah, support for Feeds makes definitely sense! PR will be very appreciated!

I think it would make more sense to create another step for the feed update if the `inputs` for this are specified and use the `swarm-cli feed update`. It seems you will have to do some hacking as the `swarm-cli identity import` requires a file to import the identity. Let me know if you need help!

**Comment by filoozom** [comment]
I ended up re-implementing the actions I needed in a new monorepo, because I wasn't quite sure it would fit in nicely here.

A successful run can be found here: https://github.com/filoozom/swarm-actions/runs/5154911654

```
Chunk Reference: e5b26537752bdb6c5632f5c3abe044fa3372c5fdb743636f84e30715187b54e4
Feed Reference: 16a49afcc8cfaef1cdfc3e682d5af6c478bedbede8e68790afd8bff8639da012
Feed Manifest: 28de3bb9d6f54b80ccdd2ba24bee5080cd2e41a868ce31e51f29b36e5b8ba366
Feed Bzz.link: https://bah5acgzafdpdxoow6vfybtg5forex3sqqdgs4qnindhddzi7fgzw4w4lunta.bzz.link
```

The objective was to be quite modular, by having the "smallest" actions possible (although I didn't really to that for `write-feed`, where the manifest is systematically created): https://github.com/filoozom/swarm-actions/blob/main/.github/workflows/test.yml
